functions/onprem/orborus/orborus.go

```diff
@@ -1072,6 +1072,34 @@ func deployK8sWorker(image string, identifier string, env []string) error {
 	env = append(env, fmt.Sprintf("IS_KUBERNETES=true"))
 	env = append(env, fmt.Sprintf("KUBERNETES_NAMESPACE=%s", os.Getenv("KUBERNETES_NAMESPACE")))
 
+	// worker resource env
```

Comment:
The SHUFFLE_WORKER_* env variables are not required on the worker. Only SHUFFLE_APP_* is needed.